components/services/bat_ads/bat_ads_impl.cc

```diff
@@ -159,7 +159,9 @@ void BatAdsImpl::GetAdsHistory(
     GetAdsHistoryCallback callback) {
   std::map<uint64_t, std::vector<std::string>> result;
 
-  auto ads_history_map = ads_->GetAdsHistory();
+  auto ads_history_map =
+      ads_->GetAdsHistory(
```

Comment:
hit: please move to end of previous line

Reply:
Done